ferrilata-devilline.IntegrationTests/Scenarios/ApiGetPitchesTests.cs

```diff
@@ -0,0 +1,84 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using ferrilata_devilline.IntegrationTests.Fixtures;
+using ferrilata_devilline.Models;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace ferrilata_devilline.IntegrationTests.Scenarios
+{
+    [Collection("BaseCollection")]
+    public class IntegrationTests
+    {
+        private readonly TestContext _testContext;
+
+        public IntegrationTests(TestContext testContext)
+        {
+            _testContext = testContext;
+        }
+
+        [Theory]
+        [InlineData("/api/pitches")]
+        public async Task PitchesApi_Should_Return401_WhenRequestHeaderAuthorizationIsMissing(string url)
+        {
```

Comment:
we need to come up with a general standard for how to name our test before any of us can commit them

Reply:
Agreed.